src/index.ts

```diff
@@ -77,6 +83,9 @@ export function publicPath(options: Options): Plugin {
     apply: "build",
     configResolved(resolvedConfig) {
       viteConfig = resolvedConfig;
+      viteConfig.excludeScriptsFilter = options.excludeScripts 
```

Comment:
I think we should append properties to `viteConfig`. Consider use `createFilter` in `processHtml` instead?

Reply:
Done.